src/accountLinks.js

```diff
@@ -0,0 +1,77 @@
+const { createLink } = require('3id-blockchain-utils')
+const utils = require('./utils')
+
+class AccountLinks {
+  constructor (ceramic, provider) {
+    this._ceramic = ceramic
+    this.provider = provider
+  }
+
+  async create (address, did, proof = null) {
+    if (!proof) {
+      if (!this.provider) {
+        throw new Error('Provider must be set to create an account link')
+      }
+      proof = await createLink(did, address, this.provider)
+    }
+    const doc = await this._ceramic.createDocument(null, 'account-link', {
+      owners: [await this._convertToCaip10(address)],
+      onlyGenesis: true
+    })
+    if (doc.state.log.length > 1) {
+      throw new Error('Account link already exists')
```

Comment:
If this is the case we should check if it already links to the correct DID.
If yes, do nothing. If no, change it.